### PR TITLE
remove PHP-FPM-stats collector

### PR DESCRIPTION
**Pull request by damacus**
# Description

remove PHP-FPM-stats collector from php containers.

### CODE REVIEW

File: opg-php-fpm-71-ppa-1604/Dockerfile [hunk 1]
```diff
@@ -4,7 +4,6 @@ ADD  docker/confd /etc/confd
 ADD  docker/logrotate.d/app /etc/logrotate.d/app
 ADD  docker/my_init.d /etc/my_init.d
 ADD  docker/nginx /etc/nginx
-ADD  docker/service/php-fpm-stats /etc/sv/php-fpm-stats
 ADD  docker/service/php-fpm /etc/sv/php-fpm
```
Comment (dragoonis) [inline opg-php-fpm-71-ppa-1604/Dockerfile]: Change ADD to COPY ?

File: opg-php-fpm-1604/Dockerfile [hunk 2]
```diff
@@ -32,12 +32,7 @@ RUN  sh /tmp/pecl_install.sh
 
 ADD  docker/nginx /etc/nginx
 
-RUN  pip install schedule statsd requests && \
-     curl -s https://raw.githubusercontent.com/ministryofjustice/php-fpm-stats-collector/master/php-fpm-stats-collector.py > /usr/local/bin/php-fpm-stats-collector.py && \
-     chmod a+x /usr/local/bin/php-fpm-stats-collector.py
-
-ADD  docker/service/php-fpm-stats /etc/sv/php-fpm-stats
-RUN  chmod a+x /etc/sv/php-fpm-stats/run && ln -s /etc/sv/php-fpm-stats /etc/service/
+RUN  pip install schedule statsd requests
```
Comment (dragoonis) [inline opg-php-fpm-1604/Dockerfile]: Are there any pip related cleanups that could be ran. Similar to how do purge composer cache or apt cache?

Comment (damacus) [reply opg-php-fpm-1604/Dockerfile]: Outside the scope of this ticket

File: opg-php-fpm-71-ppa-1604/Dockerfile (hunk 1 above)
Comment (damacus) [reply opg-php-fpm-71-ppa-1604/Dockerfile]: Outside the scope of this ticket. Agreed this isn't ideal, or what we actually want to do.

**Comment by damacus** [comment]
Confirmed working in 

https://github.com/ministryofjustice/opg-sirius/pull/67

**Pull request merged**